src/gamesList.js

```diff
@@ -0,0 +1,15 @@
+import FindTheDude from './games/findTheDude/miniGameClass.js';
+import MemoryGame from './games/memoryGame/miniGameClass.js';
+import ProtectTheCenter from './games/protectTheCenter/miniGameClass.js';
+import SpotTheDifference from './games/spotTheDifference/miniGameClass.js';
+import WaterGunRace from './games/waterGunRace/waterGunRace.js';
+
+const GAMES_LIST = [
+  WaterGunRace,
+  FindTheDude,
+  SpotTheDifference,
+  ProtectTheCenter,
+  MemoryGame,
+];
+
+export default GAMES_LIST;
```

Comment:
If you wanted to, you can do export const GAMES_LIST = [] to do the assignment and export in a single step

Reply:
Good eye.  I had initially tried that but `eslint` wasn't a fan and said I should prefer `export default` over a naked `export` (tbh I don't fully know the reasons of why, but my short-lived javascript career has been blindly following the Airbnb style guide through `eslint`)

<img src='https://user-images.githubusercontent.com/16326083/107414302-d04eab80-6adf-11eb-9683-60997eac8147.png' width=75%>

Then `export default` doesn't work inline when declaring a `const`.  So all of that led to the seemingly superfluous declare and export seen here.

If you can drop some knowledge on why preferring `export default` isn't all that `eslint` is making it out to be then I'm definitely down with the more concise `export` syntax.

Reply:
Apparently, there's some debate on if `export default` is superior or not: https://github.com/airbnb/javascript/issues/1365

Reply:
Yeah to my knowledge it's just a style convention for them to prefer export default vs the named export.
Here is a concise stack overflow link that goes over some of the differences as well
https://stackoverflow.com/a/33611943

Reply:
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export
Mozilla documentation on exports

Reply:
Yeah I think I have a good grasp on what default vs named does for the code, but I was more interested in why the style guide I'm using disallows it.  Here is a concise argument from someone who is anti-`default`: https://gitlab.com/gitlab-org/frontend/rfcs/-/issues/20#advantages-of-switching-patterns

Think I like their arguments...

Reply:
Most likely it is just from export/import conventions that existed prior. Im very much in the camp of the anti-default guy myself.

Reply:
i've converted to a named export and opened #29 to make this a project-wide preference for named exports

Reply:
Rofl what a perfect issue name

Reply:
ps ima add you as a contributor to this repo so you can do more admin-y stuff if you'd like (and commit to master if you'd like i guess....)